subprojects/packagefiles/speexdsp/libspeexdsp/meson.build

```diff
@@ -0,0 +1,53 @@
+sources = [
+  'buffer.c',
+  'fftwrap.c',
+  'filterbank.c',
+  'jitter.c',
+  'kiss_fft.c',
+  'kiss_fftr.c',
+  'mdf.c',
+  'preprocess.c',
+  'resample.c',
+  'scal.c',
+  'smallft.c',
+]
+
+cc_is_not_msvc = cc.get_id() != 'msvc'
+
+m_dep = cc.find_library('m', required : cc_is_not_msvc)
```

Comment:
https://mesonbuild.com/howtox.html#add-math-library-lm-portably

It's mandated by POSIX, it must exist but may be built into libc rather than existing as a distinct file. So this will either be found, or be unneeded, and you can simply unconditionally consider it `required: false`.

Reply:
Perfect; now the wrap is compiler-agnostic. Thanks @eli-schwartz.